wpilibc/sim/src/SensorBase.cpp

```diff
@@ -9,22 +9,21 @@
 
 #include "WPIErrors.h"
 
-const uint32_t SensorBase::kDigitalChannels;
-const uint32_t SensorBase::kAnalogInputs;
-const uint32_t SensorBase::kSolenoidChannels;
-const uint32_t SensorBase::kSolenoidModules;
-const uint32_t SensorBase::kPwmChannels;
-const uint32_t SensorBase::kRelayChannels;
-const uint32_t SensorBase::kPDPChannels;
-const uint32_t SensorBase::kChassisSlots;
+const int SensorBase::kDigitalChannels;
+const int SensorBase::kAnalogInputs;
+const int SensorBase::kSolenoidChannels;
+const int SensorBase::kSolenoidModules;
+const int SensorBase::kPwmChannels;
+const int SensorBase::kRelayChannels;
+const int SensorBase::kPDPChannels;
 
 /**
  * Check that the solenoid module number is valid.
  *
  * @return Solenoid module is valid and present
  */
-bool SensorBase::CheckSolenoidModule(uint8_t moduleNumber) {
-  return 1 <= moduleNumber && moduleNumber <= 2;  // TODO: Fix for Athena
+bool SensorBase::CheckSolenoidModule(int moduleNumber) {
+  return 1 <= moduleNumber && moduleNumber <= 63;  // TODO: Fix for Athena
```

Comment:
(1) this doesn't match the docs (valid and present? although we should probably just fix the doc...), (2) why doesn't this compare to kSolenoidModules rather than 63?